SplitTests/Integration/streaming/StreamingConnFail5xxTest.swift

```diff
@@ -0,0 +1,97 @@
+//
+// StreamingInitTest.swift
+// Split
+//
+// Created by Javier L. Avrudsky on 14-Oct-2020.
+// Copyright (c) 2020 Split. All rights reserved.
+//
+
+import XCTest
+@testable import Split
+
+class StreamingConnFail5Test: XCTestCase {
```

Comment:
is it correct the class name `StreamingConnFail5Test`? Shouldn't it be `StreamingConnFail5xxTest` like the file name?